GirafIntegrationTest/tests/test_account_controller.py

```diff
@@ -0,0 +1,355 @@
+from requests import get, post, put, delete
+import time
+from testlib import order, BASE_URL, auth, GIRAFTestCase
+
+graatand_token = ''
+graatand_id = ''
+gunnar_token = ''
+gunnar_reset_token = ''
+gunnar_username = ''
+gunnar_id = ''
+grundenberger_token = ''
+grundenberger_id = ''
+grundenberger_username = ''
+tobias_token = ''
+
+
+class TestAccountController(GIRAFTestCase):
```

Comment:
Overall I am not a very big fan of ordered tests since they rely on each other. I should be able to run a single test without running others.

From wiki:
> To isolate issues that may arise, each test case should be tested independently.

If the login test fails, some of the other tests would fail too. I recommend you to look at python mock https://docs.python.org/3/library/unittest.mock.html and consider that solution. 

This is only my opinion, others may find it unnecessary :)

Reply:
Given that these are integration tests, it sort of makes sense that they rely on each other. If you are not able to login, you should not be able to do anything that requires authentication - mocking would not reflect that.

On another note, this is simply a refactor of the old code - if this needs to be changed, perhaps it could be a separate issue?

Reply:
I see. Point taken! :)